src/App.tsx

```diff
@@ -20,12 +27,16 @@ export const App: React.FC = () => {
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                value={query}
+                onChange={(e) => {
```

Comment:
the first rule of the checklist - https://github.com/mate-academy/react_movies-list-filter/blob/master/checklist.md